### DISCUSSION TITLE
Maybe this line is useless ?

### DISCUSSION THREAD

**Discussion by FredWe** [anothermartz/Easy-Wav2Lip#51] [Q&A]
https://github.com/anothermartz/Easy-Wav2Lip/blob/752c569f1a79ed44cce8fdbd6919a38862daa586/inference.py#L639

Especially when I tried to enable wav2lip_batch_size( > 1) to accelerate inference, since this loop caused unexpected BGR problem🤨

**Comment by anothermartz** [comment]
Cool, I'll have a look into removing this and experimenting with batch size, I'm looking for performance increases now so this is perfect timing :)

**Comment by anothermartz** [comment]
I don't notice a difference in performance between batch size amounts and I prefer the per-frame updating of wav2lip_batch_size=1.

But still the removal of these lines is good as they aren't providing anything of use so thanks for pointing it out 👍
